docs/operation/solutions.rst

```diff
@@ -0,0 +1,62 @@
+Solutions Guide
+===============
+
+To deploy a Solution in a MetalK8s cluster, a utility script is provided.
+This section describes, step by step, how to deploy a Solution using this
+tool, located at the root of Metalk8s archive:
+
+  .. code::
+
+    /srv/scality/metalk8s-X.X.X/solutions.sh
```

Comment:
I think in Sphinx we can insert the right version number instead of `X.X.X`

Reply:
done